### ISSUE TITLE
Does more iteration improve detection accuracy?

### ISSUE THREAD

**Issue by 1292765944** [weiliu89/caffe#167]
I notice that in the solver.prototxt the max_iter is only 60000, How do you select the iterations. I notice that when I train the model, the loss is not stably decreased, So when should we stop training? Does more iterations improve detection accuracy? Thank you!

**Comment by ck196** [comment]
You can try it. You can plot loss graph to decide when should stop training.
I often stop training when loss does not decreased after 10000 iteration.
If your training has loss is not stably decreased, something was wrong. You should check your model, data...

**Comment by sunwooOh** [comment]
You can change the parameters(e.g. max iterations, learning rate) in $CAFFE_ROOT/examples/ssd/ssd_pascal.py.
It's in the solver_params dict, if you're still looking for it.
